pkg/network/protocols/http/testutil/pythonserver.go

```
@@ -92,6 +91,11 @@ func writeTempFile(pattern string, content string) (*os.File, error) {
 		return nil, err
 	}
 
+	err = f.Close()
+	if err != nil {
+		return nil, fmt.Errorf("failed to close file [%s]: %w", f.Name(), err)
+	}
```

we should ignore the error and to merge it in the defer

simply by
```go
defer func() { _ = f.Close() }()
```

---

or 
```
defer func() {
    if err := file.Close(); err != nil { 
         logger.Printf("Error closing file: %s\n", err)
    }
}()
```

but don't fail the function